public/js/monitor.js

```diff
@@ -528,10 +542,14 @@ function vpnBannerLogic() {
   }
 
   // Only show banner if users first language is some English-locale variant
-  if (["en", "de", "fr"].some(lang=>preferredLanguages[0].includes(lang))) {
+  if (["de", "fr"].some(lang=>preferredLanguages[0].includes(lang))) {
     vpnBannerLogic();
   }
 
+  if (["en"].some(lang=>preferredLanguages[0].includes(lang))) {
+    vpnBannerLogic({isEnglish: true});
+  }
+
```

Comment:
This logic is just reused from above. If there's a better way to do this without the `.some`, let me know.